src/content/changelog/2019-10-07-api.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "4.6.0: 2019-10-07"
+date: "2019-10-07T08:00:00-04:00"
+description: "API Changelog 4.6.0"
+changelog:
+- API
+version: 4.6.0
+---
+
+### Breaking
+
+* Removed `force` query parameter from [DELETE /object-storage/buckets/{clusterId}/{bucket}](https://developers.linode.com/api/v4/object-storage-buckets-cluster-id-bucket/#delete) endpoint.
+
+* Removed `objects`, `size`, and `region` parameters from [GET /object-storage/buckets](https://developers.linode.com/api/v4/object-storage-buckets) and [GET /object-storage/buckets/{clusterId}/{bucket}](https://developers.linode.com/api/v4/object-storage-buckets-cluster-id-bucket) endpoints.
+
+### Added
+
+* A `linode_reboot` Event action can now generate a `secondary_entity` object. A `secondary_entity` object displays information about an additional entity that is related to the Event.
+    * Use the Accounts collection to List Events [GET /account/events](/api/v4/account-events) or to View an Event [GET /account/events/{eventId}](/api/v4/account-events-event-id).
+
+* The List Managed Stats endpoint [GET managed/stats](/api/v4/managed-stats) returns a list of Managed Stats on your Account in the form of x and y data points. You can use these data points to plot your own graph visualizations. These stats reflect the last 24 hours of combined usage across all managed Linodes on your account giving you a high-level snapshot of data for `cpu`, `disk`, `swap`, `network in`, and `network out`.
+
+### Fixed
+
+* Fixed a formatting error that prevented the [filtering and sorting](https://developers.linode.com/api/v4/#filtering-and-sorting) examples within the API documentation from correctly working as presented.
+
+* Fixed an issue that allowed Linodes with IPv6 /116 pools to attempt to migrate to Next Generation Network (NGN) enabled data centers like Toronto and Mumbai. IPv6 /116 pools are not supported by NGN data centers, so these migrations would never actually progress. A Linode with IPv6 /116 pools will now be prevented from initiating a migration to an NGN data center, and the user will be prompted to contact support.
```

Comment:
datacenters is one word, not two. There are two places in here where it's separated.